main.py

```
@@ -0,0 +1,43 @@
+    logger.error("Configuration file 'project_config.yml' not found")
+    raise
+except yaml.YAMLError as e:
+    logger.error("Error parsing configuration file: %s", e)
+    raise
+
+spark = DatabricksSession.builder.profile(os.environ['DATABRICKS_PROFILE']).getOrCreate()
+
+try:
+    path = config['catalog_name'] + '.'+ config['schema_name'] + '.' + config['table_name']
+    logger.info("path_name is:" , path)
+except KeyError as e:
```

Ruff

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)